array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,13 @@
+/* exported last */
+
+// look at each element of the array.
+// if it's the last element of the array,
+// then return that element
+
+function last(array) {
+  for (var i = 0; i < array.length; i++) {
+    if (i === array.length - 1) {
+      return array[i];
+    }
+  }
```

Comment:
This loop is not necessary. You already know what index the value is at, you should just retrieve it and `return` it.